pages/favorites/index.js

```diff
@@ -0,0 +1,80 @@
+import { useContext } from "react";
+import { Favorite } from "../../pages/_app";
+import ListCard from "../../components/ListCard";
+import styled from "styled-components";
+import { Dna } from "react-loader-spinner";
+import { AiOutlineHeart } from "react-icons/ai";
+
+export default function FavoritesPage() {
+  const { favorites } = useContext(Favorite);
+
+  if (!favorites) {
+    // Wenn die Favoriten noch nicht geladen sind, zeigt ein Ladeindikator an.
+    return (
+      <div>
+        <Dna
```

Comment:
You are using the loading animation on quite some places. Maybe creating a component for it so in the future if you want to modify it you need to do it on only one places - also it saves some lines of code 🙂